nototools/unittests/font_tests.py

```diff
@@ -592,9 +592,9 @@ def setUp(self):
             if name in self.whitelist:
                 continue
             master_a[name].draw(pen_a)
-            area_a = pen_a.unload()
+            area_a = pen_a.value
```

Comment:
Yea, that's better than making a bunch of new pens. I still prefer an unload method but don't have strong feelings.